### ISSUE TITLE
Remove localisation logic from Flotilla

### ISSUE THREAD

**Issue by andchiind** [equinor/flotilla#1850]
**Describe the improvement you would like to see**
Localisation status for robots should be treated like any other robot status from ISAR. If a robot is not Offline or Delocalised, it should be assumed to be localised.

**How will this change existing functionality?**
Localisation will still be stored in Flotilla, but it will be handled as telemetry from ISAR. As a result, we should no longer set it as a value in Flotilla besides in MQTT event handler, and we should not base mission start logic on it. It a robot is Available, we should assume that it's ready to run missions. Of course, if the mission failure reason for a robot is stated as being caused by delocalisation, we do not need to try to schedule the following missions, but if we do they will fail anyways. 

In short, Localisation will be handled by ISAR. We should still send "dock" flags on missions when the queue is empty, as this is not something ISAR is aware of, but ISAR will be aware of when to undock the robot. We also no longer need to send a localisation pose in missions, as this should be handled in ISAR, but this can be relegated to another issue/PR.

**How will this improvement affect the current Threat Model?**
N/A

**Comment by tsundvoll** [comment]
Closes [#1469](https://github.com/equinor/flotilla/issues/1469)
